service/src/test/java/uk/gov/hmcts/reform/fpl/service/CaseValidatorServiceTest.java

```diff
@@ -30,6 +32,10 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 
+import static uk.gov.hmcts.reform.fpl.utils.CaseDataGeneratorHelper.createPopulatedApplicants;
+import static uk.gov.hmcts.reform.fpl.utils.CaseDataGeneratorHelper.createPopulatedChildren;
+import static uk.gov.hmcts.reform.fpl.utils.CaseDataGeneratorHelper.createRespondents;
+
 @ExtendWith(SpringExtension.class)
 class CaseValidatorServiceTest {
     private CaseValidatorService caseValidatorService;
```

Comment:
Does this need to be reset before each method? Could the `@BeforeEach` be replaced with `@BeforeAll`?

Reply:
We are asserting for different levels of errors. We have assertions for high level errors (case errors) i.e 'you have not entered respondents'. We then have assertions for lower level errors i.e in "respondents - you did not enter first name". We need a fresh CaseValidatorService for this unfortunately.